package.json

```diff
@@ -25,15 +25,15 @@
     "/src"
   ],
   "scripts": {
-    "test": "jest --config ./jest.config.js",
+    "test": "cross-env SNAPSHOT=check yarn build:dist && jest --config ./jest.config.js",
```

Comment:
is there any way we can run the test without generating the dist itself? hmm...

Reply:
AFAIK there's no way to prevent rollup writing currently. Is this a problem?

Reply:
The idea of the plugin was integration in bundler without additional configuration.

Reply:
Can this create a scenario where a merge to master can break? Thinking out loud.. i suppose not if it it updating the snapshot too

Reply:
Can we run the check as the second step of the command? (After jest)

Reply:
@alexreardon It can if PR is not based on actual master. But I think it's not so critical.